packages/headless/src/features/facets/range-facets/date-facet-set/date-format.ts

```diff
@@ -0,0 +1,12 @@
+import dayjs from 'dayjs';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+dayjs.extend(customParseFormat);
+
+export function formatDateForSearchApi(date: dayjs.Dayjs) {
```

Comment:
I didn't write this method, just moved it to prevent circular dependency

On another note, why not just always use the "official" SearchAPI date format.


(Thanks swagger!)
![Screen Shot 2021-07-16 at 2 29 33 PM](https://user-images.githubusercontent.com/4923043/125992985-4b95c759-6e89-4fbf-8d94-0fcb3e51440e.png)